### PR TITLE
Add the heartbeat code for the queues - email/sms

### PR DESCRIPTION
**Pull request by jzbahrai**
# Summary | Résumé

Heartbeat code: This function is called ever 10 seconds to check the Indox Queues for SMS and Emails. It then moves the encrypted notifications into the inflight list (please read the ADR).

- Unit tested

### CODE REVIEW

File: app/celery/scheduled_tasks.py [hunk 1]
```diff
@@ -220,3 +224,46 @@ def check_templated_letter_state():
...
+    and actually sends the sms.
+    """
+
+    # TODO: This will be instantiated from the app init.
+    sms_queue = MockQueue()
+
+    receipt_id_sms, list_of_sms_notifications = sms_queue.poll()
+
+    while list_of_sms_notifications:
+        process_inflight(receipt_id_sms, list_of_sms_notifications)
```
Comment (jzbahrai) [inline app/celery/scheduled_tasks.py:246]: make async

File: app/celery/scheduled_tasks.py [hunk 2]
```diff
@@ -220,3 +221,46 @@ def check_templated_letter_state():
                 message=msg,
                 ticket_type=zendesk_client.TYPE_INCIDENT,
             )
+
+
+@notify_celery.task(name="heartbeart-inbox-sms")
+@statsd(namespace="tasks")
+def heartbeat_inbox_sms():
+    """
+    The function acts as a heartbeat to the inbox queue. Inbox is a list in Redis that
```
Comment (jimleroyer) [inline app/celery/scheduled_tasks.py:230]:
```suggestion
    The function acts as a heartbeat to the inbox queue. Inbox is a list in the queue that
```

File: app/celery/scheduled_tasks.py [hunk 3]
```diff
@@ -220,3 +221,46 @@ def check_templated_letter_state():
...
+    is made up of notifications. The post_api will push all the notifications to Inbox.
```
Comment (jimleroyer) [inline app/celery/scheduled_tasks.py:231]: Let's avoid mentioning the inbox and refer to the queue simply, as this is an internal implementation detail to the `RedisQueue` and that could change in the future.

File: app/celery/scheduled_tasks.py [hunk 4]
```diff
@@ -220,3 +221,46 @@ def check_templated_letter_state():
...
+
+@notify_celery.task(name="heartbeart-inbox-email")
+@statsd(namespace="tasks")
+def heartbeat_inbox_email():
+    """
+    The function acts as a heartbeat to the inbox queue. Inbox is a list in Redis that
+    is made up of notifications. The post_api will push all the notifications to Inbox.
+    The heartbeat with check the inbox and move the notifications to the inflight list
+    accordingly. The heartbeat will then call the job that sends the list to the DB
+    and actually sends the email.
```
Comment (jimleroyer) [inline app/celery/scheduled_tasks.py:255]: Same comment would apply here with the inbox, where we should refer to the queue.

**Review by jimleroyer — APPROVED** [review]
LGTM -- minor fixes around the documentation.

**Pull request merged**